src/xdocs/config_imports.xml

```diff
@@ -997,6 +997,64 @@ public class InputIllegalImport { }
         </source>
 
         <p>
+          In the next examples usage of <code>strategyOnMismatch</code> property is shown.
+          This property defines strategy in a case when no matching allow/disallow rule was found.
+          <code>strategyOnMismatch</code> property is attribute of <code>import-control</code> and <code>subpackage</code> tags.
```

Comment:
should be "Property <code>strategyOnMismatch</code> " . It is better to not start sentence from code block.